uftrace.c

```diff
@@ -860,7 +860,9 @@ int main(int argc, char *argv[])
 	setup_color(opts.color);
 	setup_signal();
 
-	if (opts.mode == UFTRACE_MODE_RECORD || opts.mode == UFTRACE_MODE_RECV)
+	if (opts.mode == UFTRACE_MODE_RECORD
+	    || opts.mode == UFTRACE_MODE_RECV
+	    || opts.nop)
```

Comment:
It's just a coding style issue - I prefer putting the operator at the end of line.  And I think it'd be better splitting nop check into another line like below:

```
if (opts.mode = ... RECORD || ... RECV)
    opts.use_pager = false;
if (opts.nop)
    opts.use_pager = false;